package-lock.json

```diff
@@ -13745,6 +13748,12 @@
         "postcss": "^7.0.14"
       }
     },
+    "icu4c-data": {
+      "version": "0.64.2",
+      "resolved": "https://registry.npmjs.org/icu4c-data/-/icu4c-data-0.64.2.tgz",
+      "integrity": "sha512-BPuTfkRTkplmK1pNrqgyOLJ0qB2UcQ12EotVLwiWh4ErtZR1tEYoRZk/LBLmlDfK5v574/lQYLB4jT9vApBiBQ==",
+      "dev": true
+    },
```

Comment:
Is this change intended?

Reply:
not intended, although it has been being added automatically when running the check-styles task. Not sure if it is a dependency that should now be added to the lock file.

Reply:
We usually avoid these isolated changes only on the package-lock files unless there is an explicit dependency change/upgrade. We can revert this

Reply:
ok!

Reply:
done!